CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## 2020.9.1 (10 September 2020)
+
+Notable changes:
+-   Bug Fix: Addressed memory and cpu issues a number of users had with indexer but setting it off by default 
```

Comment:
but -> by

Reply:
so previous line: Pylance will now index libraries and unopened files at startup to provide auto-import completions even for variables that have not been fully analyzed. This index is also used to improve the performance of the workspace symbols search.


- Addressing memory and cpu issues a number of users had by no longer indexing libraries and unopened files at startup.  This will revert auto-import completions and workspace symbols performance to previous levels.